config/environments/development.rb

```diff
@@ -1,4 +1,4 @@
-Rails.application.configure do
+Rails.application.configure do # rubocop:disable Metrics/BlockLength
```

Comment:
exclude all configs for this metric instead

Reply:
done